Packs/CommonDashboards/ReleaseNotes/1_4_3.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: UseCaseAdoptionMetrics
+
+New: the dashboard provide a high level overview on the usage of XSOAR. It contains datapoints and metrics on enriched and observed threat intelligence, playbook and automation executions, and data sources being ingested into xsoar.
```

Comment:
```suggestion
New: The dashboard provide a high level overview on the usage of Cortex XSOAR. It contains datapoints and metrics on enriched and observed threat intelligence, playbook and automation executions, and data sources being ingested into Cortex XSOAR.
```
But isn't this a script?  Not a dashboard.  The description does not really make sense.